src/riscv/lib/src/machine_state/instruction/constructors.rs

```diff
@@ -120,7 +120,7 @@ impl Instruction {
         }
     }
 
-    /// Create a new [`Instruction`] with the appropriate [`super::ArgsShape`] for the [`OpCode::Neg`].
+    /// Create a new [`Instruction`] for the [`OpCode::Neg`].
```

Comment:
```suggestion
    /// Create a new [`Instruction`] for [`OpCode::Neg`].
```

Reply:
Remove the word in [this change](https://github.com/tezos/riscv-pvm/compare/7761e794af17d2e6cb7eb977a39caa04ae1d67a6..6b1bffa5a9f2dd78219fcf83ea0a7a6715051d44).